Jumpit/JumpitAPI.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  JumpitAPI.swift
+//  Jumpit
+//
+//  Created by 이지훈 on 5/22/24.
+//
+
+import Foundation
+import Moya
+
+enum JumpitAPI {
+    case postResume(data: ResumeData)
+    case fetchMyResume(userId: String)
+    case toggleResumePrivacy(resumeId: String)
+    case searchPositions(keyword: String)
+    case searchPositionsFiltered(keyword: String, categories: [Int])
+    case fetchPositionDetail(positionId: String)
+}
+
+extension JumpitAPI: TargetType {
+    var baseURL: URL {
+        guard let urlString = Bundle.main.object(forInfoDictionaryKey: "BaseURL") as? String,
+              let url = URL(string: urlString) else {
+            fatalError("BaseURL not set in plist for this environment")
+        }
+        return url
+    }
+
+    var path: String {
+        switch self {
+        case .postResume:
+            return "/resumes"
+        case .fetchMyResume(let userId):
+            return "/resumes/\(userId)"
+        case .toggleResumePrivacy(let resumeId):
+            return "/resumes/\(resumeId)"
+        case .searchPositions, .searchPositionsFiltered:
+            return "/positions"
+        case .fetchPositionDetail(let positionId):
+            return "/positions/\(positionId)"
+        }
+    }
+
```

Comment:
```suggestion
extension JumpitAPI: JumpitAPIEndpoint {
    var domain: JumpitAPIDomain {
        switch self {
        case .postResume, .fetchMyResume, .toggleResumePrivacy:
            return .resumes
        case .searchPositions, .searchPositionsFiltered, .fetchPositionDetail:
            return .positions
        }
    }

    var urlPath: String {
        switch self {
        case .postResume:
            return ""
        case .fetchMyResume(let userId), .toggleResumePrivacy(let resumeId):
            return "/\(userId)"
        case .searchPositions, .searchPositionsFiltered:
            return ""
        case .fetchPositionDetail(let positionId):
            return "/\(positionId)"
        }
    }
``` 
<2번 방식> 파일하나에 두개의 분기처리를 모두 해 주실 경우 이런식으로 바꾸면 적어주신 JumpitAPIDomain을 활용할 수 있을 것 같습니다!